markdown/chapters/meandeck/basics.md

```diff
@@ -4,59 +4,110 @@ order: 0
 title: The Basics
 ---
 
-This section explains the basic Doomsday piles that attempt to end the game
-directly with a {{Thassa's Oracle}} trigger. While older versions of Meandeck
-used {{Laboratory Maniac}} as the primary win-condition, the introduction of TO
-allows us to build piles that are more streamlined than ever.
+The goal of all Magic decks is to win the game. {{Doomsday}} aims to do so by casting
+its namesake card and constructing a *pile* of 5 cards tailored to the current
+game state. While it would be nigh impossible to enumerate every game state and 
+pile variation, these chapters are aimed to show you the basic concepts behind casting
+and winning with Doomsday. There is one primary win condition that is used in this
+version of the deck to win with and that is via {{Thassa's Oracle}}.
+
+## Pile Construction Steps
 
 A *pile* in this context is what we use to describe the stack of five cards that
-comprise your library after the resolution of Doomsday.
+comprise your library after the resolution of Doomsday. This is terminology that 
+will be used throughout the Wiki.
 
-As a general rule, in order to win you need to:
+There are three steps that need to occur to enable you to win (at least most of the time).  
+These are as follows:  
 
 1. Resolve Doomsday
-1. Draw into your pile
-1. Consume enough cards out of the pile so that TO's trigger wins you the game
+1. Draw into your pile enough so that the Oracle's trigger can win you the game
+1. Resolve Thassa's Oracle 
+
+Out these steps, we will start by detailing how to achieve steps 2. and 3. as these are
+the first steps that are unique to the Doomsday archetype. Other decks like Omnitell have
+similar transferable skills in order to resolve a specific key spell however it is the
+*pile* construction that is often the trickiest element of playing the deck for newer
+players of the archetype.
+
+The first consideration to make is how you will *dig* into the pile.  
+We will use the phrase *dig* to mean to initially draw into the five cards you have stacked
+with Doomsday.  
+The most common way to *dig* into your pile is via a *cantrip* effect. This is a spell or
+ability which draws a card upon resolution. Typically this is provided by one of the commonly
+played {U} filter effects like {{Preordain}} or a free cycle effect like {{Street Wraith}}.
+If you are not under pressure you can also simply pass the turn to the opposing player and use
+your next turn's draw step in order to *dig* into the *pile*. When this is performed it is
+simply know as a *pass the turn pile* or *PTT pile*. 
```

Comment:
i believe it should known instead of know

Reply:
Fixed.